libs/libvtrutil/src/vtr_vec_id_set.h

```diff
@@ -2,6 +2,7 @@
 #define VTR_SET_H
 
 #include <vector>
+#include <algorithm>
```

Comment:
Why was this added?

Reply:
After moving around the code, the compiler gave me an error saying ```error: 'sort' is not a member of 'std'``` in L79 of this file. That's why I added it.